resource-adapter/resource-adapter-deployment/diskHome/xadisk.log_1463833983405

```diff
@@ -0,0 +1,6 @@
+Apr 27, 2016 7:11:04 PM : Successfully booted the XADisk instance.
```

Comment:
No

--
Sent from my iPhone


On May 21, 2016, at 9:32 AM, George Gastaldi <notifications@github.com>
wrote:

In
resource-adapter/resource-adapter-deployment/diskHome/xadisk.log_1463833983405
<https://github.com/wildfly-swarm/wildfly-swarm-examples/pull/76#discussion_r64134640>
:

> @@ -0,0 +1,6 @@
> +Apr 27, 2016 7:11:04 PM : Successfully booted the XADisk instance.

Are these log files supposed to exist in the Git repository?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/wildfly-swarm/wildfly-swarm-examples/pull/76/files/b4bdc755861e244553c7d0d149893d7e23ce1ddb#r64134640>